getting-started/host_security_policy_specification.md

```diff
@@ -70,6 +70,8 @@ spec:
   action: [Audit|Block] (Block by default)
 ```
 
+**Please Not that for system calls monitoring, we only support audit action not matter what the value of action is**
```

Comment:
**Please note that for system calls monitoring we only support audit action no matter what the value of action is**

Reply:
Let's use the Github Formatted Notes here

> **Note** Your note

You can see how it's formatted specially here.
Can leverage with
```
> **Note** Your note
```